lib/engine.js

```diff
@@ -30,6 +30,9 @@ module.exports = function container (get, set, clear) {
       console.error('error: could not find product "' + s.product_id + '"')
       process.exit(1)
     }
+    if ((so.mode === 'live' || so.mode === 'paper') && selector_parts[0] === 'cexio') {
+      s.exchange.setFees({asset: s.asset, currency: s.currency})
+    }
```

Comment:
Is there any way to prevent this from ending up as an exchange-specific hack in `lib/engine.js`?

Reply:
Was initially considering adding the setFees: function() to every exchange (as a pseudo call doing nothing for now except for CEX) but didn't want to change to many files.

But if this gets adopted for other exchanges i don't see a reason to not do it a cleaner/less hackish/none exchange-specific way.

Your call :)

Reply:
On a second though i could add a new exchange property (called dynamicFees or whatever) and check for it in engine.js

Let me update the PR

Reply:
Done